ui/app/components/providers/actors-relationships.hbs

```diff
@@ -0,0 +1,3 @@
+{{yield
```

Comment:
The provider pattern here is very cool! What do you think about making the folder structure here as a child of `/evaluation-sidebar`? I think this might benefit from the context.